content/start/core-concepts/head-and-metadata.mdx

```diff
@@ -1,17 +1,178 @@
-<Title>Head and metadata</Title>
+import { Aside } from "~/components/FrameworkAside";
 
-## Populating the `<head>` of your document
+# Head and metadata
 
-### Add tags in `root.tsx`
+Normally when we are building UIs for our apps, we are dealing with DOM elements that are going to be rendered in the `body` of the `document`. But there are cases where we want to customize what's going to be rendered in the `head` of the `document`. The common elements used in the `head` are:
 
-### Use `solid-meta` inside component tree
```

Comment:
Is it correct to say `solid-meta` when we are really importing from `solid-start`?